config/locales/candidate_mailer.yml

```diff
@@ -81,3 +81,6 @@ en:
         course_full: "%{course_name} at %{provider_name} became full while %{candidate_name} was awaiting references"
         location_full: "%{course_name} at %{provider_name} became full at %{location} while %{candidate_name} was awaiting references"
         study_mode_full: "%{study_mode} places for %{course_name} at %{provider_name} became full while %{candidate_name} was awaiting references"
+    ucas_match_initial_email:
+      duplicate_applications:
+        subject: 'Action required: It looks like you submitted a duplicate application'
```

Comment:
Should this `It` be capitalized?

Reply:
Yes. All subject lines mentioned [in the email document](https://docs.google.com/document/d/1s5ql4jNUUr3QDPUQYWImkZR6o8upvutrjOEuoZ0qqTE/edit) follow this pattern.

Reply:
Ah. Feels a bit odd to me — getting content design advice!

Reply:
Lower case is correct: https://ukgovernmentdfe.slack.com/archives/CPH8J9G65/p1606217719066500?thread_ts=1606212747.059500&cid=CPH8J9G65

Reply:
@duncanjbrown updated.